From 989ad77e979c7f8880d07a3e1e7f55f8e1fad8b9 Mon Sep 17 00:00:00 2001
From: Bandana <aurelende@protonmail.com>
Date: Tue, 14 Apr 2020 10:35:29 -0400
Subject: [PATCH 12/15] diag knockback fix

Signed-off-by: Bandana <aurelende@protonmail.com>
---
 Intersect.Server/Entities/Entity.cs | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Intersect.Server/Entities/Entity.cs b/Intersect.Server/Entities/Entity.cs
index 0351c6fa..34616ced 100644
--- a/Intersect.Server/Entities/Entity.cs
+++ b/Intersect.Server/Entities/Entity.cs
@@ -1445,7 +1445,8 @@ byte projectileDir
             }
 
             //If there is a knockback, knock them backwards and make sure its linear (diagonal player movement not coded).
-            if (projectile.Knockback > 0 && projectileDir < 4)
+            //Diagonal movement coded with diagonal patch
+            if (projectile.Knockback > 0 && projectileDir < 8)
             {
                 var dash = new Dash(target, projectile.Knockback, projectileDir, false, false, false, false);
             }
-- 
2.25.0.windows.1

